src/mqtt.cpp

```diff
@@ -107,6 +108,43 @@ void mqttmsg_callback(MongooseString topic, MongooseString payload) {
       divertmode_update(newdivert);
     }
   }
+  else if (topic_string == mqtt_topic + "/charge_current/set")
+  {
+    int newchargecurrent = payload_str.toInt();
+    DBUGF("Set charge_current: %d", newmode);
+    //using override, the only interest of this is having priority above everything else.
+    //We have a confusion in web interface to solve as details of the override are not displayed
```

Comment:
The GUI could be updated to show that the EVSE is being controlled be an external source and use the `/claims` API to get the info

Reply:
Agreed, that is the way. I have not gave a look yet the the javascript gui part as it's embedded in .h file, looks unreadable :)